sdcm/sct_config.py

```diff
@@ -1775,6 +1785,18 @@ def __init__(self):
                 raise ValueError("For PasswordAuthenticator authenticator authenticator_user and authenticator_password"
                                  " have to be provided")
 
+        # 13) validate stress and prepare duration:
+        if stress_duration := self.get('stress_duration'):
+            try:
+                self['stress_duration'] = abs(int(stress_duration))
+            except ValueError:
+                raise ValueError(f'Configured stress duration for generic test duratinon have to be \
```

Comment:
The message is not correct. Your code will transform negative int value into positive using the `abs` function.

Reply:
in cases the stress_duration will be set as negative int (-50) it will be converted to positive without errors, but if value will some string, then exception will inform that it should be set as positive integer

Reply:
The `string` value will be transformed into the `integer` value by the `int()` function before processing by the `abs()` one.

Reply:
remove all this validation, @vponomaryov is correct that it's already gonna happen automatically